README.md

```diff
@@ -220,6 +220,39 @@ physeq
 ## phy_tree()    Phylogenetic Tree: [ 759 tips and 757 internal nodes ]
 ```
 
+## Creating a TreeSummarizedExperiment (TSE) Object
+
+A wrapper function called `qza_to_tse()` is provided which links multiple `read_qza()` calls together to create a TSE object for subsequent analysis. You may find some useful tutorials on :
+[OMA tutorials](https://jserizay.com/OMA/docs/devel/). An example usage is shown below:
```

Comment:
this is a wrong URL. The current one is https://microbiome.github.io/OMA/docs/devel/